internal/file/file.go

```diff
@@ -131,3 +132,8 @@ func ParseMediaType(content []byte) (string, error) {
 	}
 	return manifest.MediaType, nil
 }
+
+// IsImageManifest checks whether a manifest is an image manifest.
+func IsImageManifest(mediaType string) bool {
+	return mediaType == docker.MediaTypeManifest || mediaType == ocispec.MediaTypeImageManifest
+}
```

Comment:
This function is not related to file. It is better to put it back to the `descriptor` package.

Reply:
Even consider
```go
func IsImageManifest(desc ocispec.Descriptor) bool
```